.gitignore

```diff
@@ -35,3 +35,6 @@ commands.log
 .vscode/settings.json
 
 .pytest_cache/
+
+system_tests.zip
+system_tests/ni-nimi-python-*
```

Comment:
I'm confused on why this is needed all of a sudden.

Reply:
I brought this up in the dev chat a while back because I ran the system tests in my dev system, and hundreds of these files were created in the repo. Shreyas suggested I include them in .gitignore:

![image](https://user-images.githubusercontent.com/80292064/115940117-02787400-a466-11eb-9353-5316589c58a4.png)

Reply:
Can you please describe the workflow that created the `system_tests` folder and zip file.

Reply:
From what I recall, all I did was install the built version of the API and then ran the tests locally, using the following command:

pytest -v src\nidcpower\system_tests\test_system_nidcpower.py

I can't confirm it now because for the last day or two, I haven't been able to run the tests locally -- I'm getting an "Urecoverable Failure" (-1074135040) error in niDCPower_InitializeWithChannels.

Nevertheless, if you or Marcos are uncomfortable including the .gitignore change in this PR, I don't mind removing it and waiting until I can reproduce it again.

Reply:
Fair enough, I've removed that edit. If and when I see it happen again, I'll create a new PR with a description of the workflow.

Reply:
Single purpose PRs are best anyway.